website/content/docs/_index.md

```diff
@@ -218,4 +219,42 @@ To make `wash find`'s filtering less tedious and better performing, entries can
 
 NOTE: _All_ attributes are optional, so set the ones that you think make sense. For example, if the `mode` or `size` attributes don't make sense for your entry, then feel free to ignore them. However, we recommend that you try to set the `meta` attribute when you can to take advantage of metadata filtering.
 
-NOTE: We plan on adding more attributes depending on user feedback (e.g. like `state` and `labels`). Thus if you find yourself metadata-filtering on a common property across a bunch of different entries, then please feel free to file an issue so we can consider adding that property as an attribute (and as a corresponding `wash find` primary).
\ No newline at end of file
+NOTE: We plan on adding more attributes depending on user feedback (e.g. like `state` and `labels`). Thus if you find yourself metadata-filtering on a common property across a bunch of different entries, then please feel free to file an issue so we can consider adding that property as an attribute (and as a corresponding `wash find` primary).
+
+### Entry Schemas
+
+Entry schemas are a type-level overview of your plugin's hierarchy. They enumerate the kinds of things your plugins can contain, including what those things look like. For example, a Docker container's schema would answer questions like:
+
+* Can I create multiple Docker containers?
+* What's in a Docker container's metadata?
+* What Wash actions does a Docker container support?
+* If I `ls` a Docker container, what do I get?
+
+These questions can be generalized to any Wash entry.
+
+Entry schemas are a useful way to document your plugin's hierarchy without having to maintain a README. Users can view your hierarchy via the `stree` command. For example, if you invoke `stree docker` in a Wash shell (try it!), you should see something like
+
+```
+docker
+├── containers
+│   └── [container]
+│       ├── [fs]
+│       │   ├── [file]
```

Comment:
Interesting that the order of `file/dir` is swapped in `dir`. It looks kind of weird.

Reply:
Ah yeah, that's because `apitypes.EntrySchema` stores the children in a `map[string]*EntrySchema`. I think I can fix that to be an array instead so insertion order's preserved.